src/firefly/java/edu/caltech/ipac/firefly/server/servlets/ServerStatus.java

```diff
@@ -119,11 +105,19 @@ private static void displayCacheInfo(PrintWriter writer, CacheManager cm) {
     private static void showDatabaseStatus(PrintWriter writer) {
         writer.println("DATABASE INFORMATION");
         writer.println("--------------------");
-        writer.println("Open: " + BaseDbAdapter.getDbInstances().size());
-        writer.println("Details: idle time is in (mm:ss)");
-        Collections.unmodifiableCollection(BaseDbAdapter.getDbInstances().values()).stream()
+        writer.println(String.format("Open: %d    CHECK_INTVL: %ds    MAX_ROWS: %d    MAX_IDLE: %dm",
+                        DbAdapter.getAdapter().getDbInstances().size(), DbAdapter.CLEANUP_INTVL/1000, DbAdapter.MAX_MEMORY_ROWS, DbAdapter.MAX_IDLE_TIME/1000/60));
+        writer.println("Idled   Age     Tables  Rows      File Path");
+        writer.println("------  ------  ------  --------  ---------");
+        Collections.unmodifiableCollection(DbAdapter.getAdapter().getDbInstances().values()).stream()
                     .sorted((db1, db2) -> Long.compare(db2.getLastAccessed(), db1.getLastAccessed()))
-                    .forEach((db) -> writer.println(String.format("\tidled: %2$tM:%2$tS %s", db.getDbFile().getPath(), System.currentTimeMillis() - db.getLastAccessed())));
+                    .forEach((db) -> writer.println(String.format("%4$tM'%4$tS\"  %5$tM'%5$tS\"  %6d  %8d  %s",
+                                                        db.getTblCount(),
+                                                        db.getRowCount(),
+                                                        db.getDbFile().getPath(),
+                                                        System.currentTimeMillis() - db.getLastAccessed(),
+                                                        System.currentTimeMillis() - db.getCreated()
+                    )));
```

Comment:
I am staring at this println and do not see how the order of the arguments matches the format or the header line above. :) Oh, `4$` and `5$` stand for the argument indices! 

Anyway, I would use ':' to separate minutes and seconds. I think the notation you used is for angles, not for time.

Also, where would I see this printout? How do I check the db status?

Reply:
Status page link is posted in PR.  http://localhost:8080/firefly/admin/status

I want to denote duration, not time.  How do I say it's been idled for 3 minutes and 20 seconds?

Reply:
I am not an expert in English notation. :) I just don't remember seeing ' and '' for time anywhere. I was curious though to google it up. Looks like this notation was used for duration (elapsed time) before, but it's hardly used now:
https://english.stackexchange.com/questions/114205/english-notation-for-hour-minutes-and-seconds 
https://www.dailywritingtips.com/how-to-refer-to-time/